### PR TITLE
Add a FAQ

### PR DESCRIPTION
**Pull request by pyrech**
Please review carefully as there is probably some remaining typo :hugs:

### CODE REVIEW

File: templates/content/faq.html.twig [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                <h1 class="content-head is-center">Frequently asked questions</h1>
+
+                <ul>
+                    <li id="data-collect">
+                        <h2>
+                            <a href="#data-collect">What do you do with the data you collect?</a>
+                        </h2>
+                        <p>
+                            <em>Short answer:</em>
+                            Nothing :)
```
Comment (damienalexandre) [inline templates/content/faq.html.twig]:
```suggestion
                            Nothing 😚
```

File: templates/content/faq.html.twig [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                        </p>
+                        <p>
+                            <em>Detailled answer:</em>
+                            We receive only the data mandatory to start the Secret Santa:
+                            the list of users and usergroups in your organization. Once the private messages have been sent,
+                            we forget everything! In fact, we do not even have a database ;)
```
Comment (damienalexandre) [inline templates/content/faq.html.twig]:
```suggestion
                            we forget everything! In fact, we do not even have a database 😮
```

File: templates/content/faq.html.twig [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                        </p>
+                    </li>
+                    <li id="avoid-association">
+                        <h2>
+                            <a href="#avoid-association">Can I avoid someone to receive a gift from someone else?</a>
```
Comment (damienalexandre) [inline templates/content/faq.html.twig]:
```suggestion
                            <a href="#avoid-association">Can I make sure someone will be matched or not matched with someone else?</a>
```

File: templates/content/faq.html.twig [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+                            An alternative could be to launch 2 Secret Santa with two groups of people.
+                        </p>
+                    </li>
+                    <li id="hall-of-fame">
+                        <h2>
+                            <a href="#hall-of-fame">How can I add my company / organization to your Hall of Fame?</a>
+                        </h2>
+                        <p>
+                            Simply drop us an email at <a href="mailto:hello@secret-santa.team">hello@secret-santa.team</a>
+                            with the name, url and logo of the entity you want to add.
```
Comment (damienalexandre) [inline templates/content/faq.html.twig]:
```suggestion
                            with the name, url and logo of the entity you want to add. It's free but if you want to tweet or share our app to the world, you are more than welcome.
```

File: templates/content/faq.html.twig [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+                        </p>
+                    </li>
+                    <li id="big-team">
+                        <h2>
+                            <a href="#big-team">Will you app work with team with more than 500+ people?</a>
```
Comment (damienalexandre) [inline templates/content/faq.html.twig]:
```suggestion
                            <a href="#big-team">Will the app work with team larger than 500 people?</a>
```

File: templates/content/faq.html.twig [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+                        </h2>
+                        <p>
+                            There is a 2 things to understand here. The first step in our application is to retrieve
+                            all the users on your Slack workspace / Discord server - yes we retrieve all the users to allow
+                            you to select the ones that will participate to your Secret Santa.
+                        </p>
+                        <p>
+                            When your team is really big, Slack or Discord APIs can fail to send us all the users in a
+                            reasonable duration. That means you will no be able to start a Secret Santa at the moment. A
```
Comment (damienalexandre) [inline templates/content/faq.html.twig]:
```suggestion
                            reasonable duration. That means you will no be able to start a Secret Santa. A
```

File: config/routes.yaml [hunk 7]
```diff
@@ -18,6 +18,10 @@ privacy_policy:
     path:     /privacy-policy
     defaults: { _controller: JoliCode\SecretSanta\Controller\ContentController::privacyPolicy }
 
+faq:
+    path:     /faq
+    defaults: { _controller: JoliCode\SecretSanta\Controller\ContentController::faq }
```
Comment (lyrixx) [inline config/routes.yaml]: You can use TemplateController
https://symfony.com/doc/current/templating/render_without_controller.html

**Pull request merged**